reference/5.1/Microsoft.PowerShell.Utility/Write-Host.md

```diff
@@ -64,8 +66,8 @@ Write-Host (2,4,6,8,10,12) -Separator ", +2= "
 2, +2= 4, +2= 6, +2= 8, +2= 10, +2= 12
 ```
 
-This command displays the even numbers from two through twelve.
-The Separator parameter is used to add the string `, +2= (comma, space, +, 2, =, space)`.
+This command displays the even numbers from two through twelve. The Separator parameter is used to
+add the string `, +2= (comma, space, +, 2, =, space)`.
```

Comment:
Formatting problem: I suggest:

```none
`, +2= ` (comma, space, `+`, `2`, `=`, space).
```
to have it render as:

> `, +2= ` (comma, space, `+`, `2`, `=`, space).

Also, with respect to styling parameter-name references: "Separator" isn't styled here.
Personally, I'd find the styling `-Separator`  most useful (even though the `-` isn't technically part of the parameter, it makes the reference readily recognizable).